src/API/Base.php

```diff
@@ -399,108 +465,21 @@ public static function get_merchant( $merchant_id ) {
 
 
 	/**
-	 * Creates a merchant for the authenticated user or returns the existing one.
+	 * Creates a merchant for the authenticated user or updates the existing one.
```

Comment:
```suggestion
	 * Creates a merchant for the authenticated user or updates the existing one. On success the Merchant ID is returned instead of the full merchant object. 
```